Qualisys/DemoAssets/UI/ServerButton.cs

```diff
@@ -27,7 +27,7 @@ void OnClick()
 
     IEnumerator Connect()
     {
-        RTClient.GetInstance().StartConnecting(response.IpAddress, -1, true, true, false, true, false, true);
+        RTClient.GetInstance().StartConnecting(response.IpAddress, -1, true, true, false, true, false, true, true);
```

Comment:
I think it is high time this function takes a parameter struct instead of all these booleans.

Reply:
There are many functions that take almost the same parameters. Would look better if they were using the same struct for at least the parameters they are sharing.

Reply:
Added issue.
[Selecting streaming components is unergonomic. #30](https://github.com/qualisys/Qualisys-Unity-SDK/issues/30)